### DISCUSSION TITLE
Question - forcing a LXC to go through a vpn

### DISCUSSION THREAD

**Discussion by Koliera** [community-scripts/ProxmoxVE#242] [Q&A]
Hi there,
I pretty much have moved over to LXCs except my OMV VM. I wanted to ask is there a way to run the LXCs through a VPN similar to how Gluetun (https://github.com/qdm12/gluetun) can handle everything that is listed in a docker stack?
Thanks.

**Comment by Mellowlynx** [comment]
You mean inside to out, right?
It might be possible, but I would leave it a VM, to keep your setup simple.

**Comment by Koliera** [comment]
Correct, as in an LXC would be forced to connect to the internet through a VPN.

**Comment by rekenaar00** [comment]
Hi
Maybe I overcomplicated, but I defined a SDN in proxmox. 
So I have my normal network, vmbr0, for most containers.
And then the software network, vxnet, for QB & *arr.
Then I have an Openwrt VM, with access to both networks. It is configured to handle dhcp on vxnet. It is configured to autoconenct to VPN,
Then my QB & *arr lxc's only have access to vxnet for internet connecting through VPN.
Also some port forwards on Openwrt  to allow access to, for example QB, from browser on normal lan.

Stuggled with this a long time ago, so not sure I will be able to assist from scratch, but maybe it can send you in a direction.

I think [this was the starting point](https://forum.proxmox.com/threads/is-there-a-cluster-bridge-or-cluster-level-networking.124309/#post-543900)

And [this from Nord](https://support.nordvpn.com/hc/en-us/articles/20340177222289-OpenWrt-setup-with-NordVPN) also helped a lot,
